cli/pipeline/pipeline_definition_test.go

```diff
@@ -10,7 +10,7 @@ func TestParsePipeline(t *testing.T) {
 	if err != nil {
 		panic(err.Error())
 	}
-	pipeline := parsePipeline(data)
+	pipeline := ParsePipeline(data)
```

Comment:
🤔. Thanks for fixing this! Any idea how this was this passing through CI before?

Reply:
Not sure if there is a CI, [release](https://github.com/deliveroo/paddle#release) is manual